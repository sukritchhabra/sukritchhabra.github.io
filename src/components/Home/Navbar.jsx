import React from 'react';

import './Navbar.scss';

const Navbar = () => (
  <nav className="navbar">
    <h1>Sukrit Chhabra</h1>
  </nav>
);

export default Navbar;
