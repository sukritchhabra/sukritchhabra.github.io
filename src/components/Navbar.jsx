import React from 'react';

import 'styles/Navbar.scss';

const Navbar = () => (
  <nav className="navbar">
    <h1>Sukrit Chhabra</h1>
  </nav>
);

export default Navbar;
