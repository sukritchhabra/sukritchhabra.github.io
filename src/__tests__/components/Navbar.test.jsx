import React from 'react';
import { shallow } from 'enzyme';
import Navbar from 'components/Navbar';

describe('Navbar.jsx', () => {
  const wrapper = shallow(<Navbar />);

  test('renders', () => {
    expect(wrapper.exists()).toBeTruthy();

    const navbar = wrapper.find('h1');
    expect(navbar).toHaveLength(1);
    expect(navbar.text()).toEqual('Sukrit Chhabra');
  });

  test('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
