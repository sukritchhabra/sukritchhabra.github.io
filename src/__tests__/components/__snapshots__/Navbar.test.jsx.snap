// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar.jsx matches snapshot 1`] = `
<nav
  className="navbar"
>
  <h1>
    Sukrit Chhabra
  </h1>
</nav>
`;
